projects/backend/fronts.ts

```diff
@@ -62,15 +62,16 @@ export const parseCollection = async (
                 (furniture && furniture.kicker) || article.kicker || '' // I'm not sure where else we should check for a kicker
             const headline =
                 (furniture && furniture.headlineOverride) || article.headline
-            const trail =
+            const trail = (
                 (furniture && furniture.trailTextOverride) ||
                 article.trail ||
                 ''
+            ).replace(/<\/?[^>]+>/gi, '')
```

Comment:
Might be worth using a library for this instead of a regex?

Reply:
we got `striptags` already in! i'd use that. We also have a `cleanupHtml` function that will preserve linebreaks

Reply:
Thanks - have updated.